contracts/test/MockCToken.sol

```diff
@@ -0,0 +1,67 @@
+pragma solidity ^0.8.0;
+
+import "../interfaces/compound/ICToken.sol";
+import "../utils/WadRayMath.sol";
+import "../utils/Printer.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract MockCToken is ICToken, ERC20 {
+    using WadRayMath for uint256;
+    address internal _cToken;
+    address internal _underlyingAsset;
+    uint256 internal _rate;
+
+    function balanceOfUnderlying(address owner) external returns (uint256) {
+        return (balanceOf(owner) * exchangeRateCurrent()) / 1e18;
+    }
+
+    constructor(
+        address underlyingAsset,
+        string memory name,
+        string memory symbol
+    ) ERC20(name, symbol) {
+        _underlyingAsset = underlyingAsset;
+    }
+
+    function setExchangeRate(uint256 rate) external {
+        _rate = rate;
+    }
+
+    function exchangeRateCurrent() public override returns (uint256) {
+        return _rate;
+    }
+
+    function exchangeRateStored() public view override returns (uint256) {
+        return _rate;
+    }
+
+    function underlying() external view override returns (address) {
+        return _underlyingAsset;
+    }
+
+    function redeemUnderlying(uint256 redeemAmount)
```

Comment:
why does the redeem function not return anything?

Reply:
0 = success return code. The FCM wasn't checking this return code anyway but I've just changed that.

Quite possible that this mock will need more logic eventually as we write more tests, but I think we should just add logic as required by our tests.

Reply:
agreed